tests/ncc/ncc_checkrepos.pm

```diff
@@ -0,0 +1,18 @@
+use base "x11test";
+use testapi;
+
+sub run() {
```

Comment:
please have that test start with sle11 too - it's doubtful we will ever have a ncc for another product